core/src/components/item/test/fill/index.html

```diff
@@ -78,7 +78,7 @@
               </ion-item>
             </ion-col>
             <ion-col>
-              <ion-item class="ion-invalid" fill="solid">
+              <ion-item class="ion-valid" fill="solid">
```

Comment:
I noticed there were no "valid" examples, so I changed extra invalid examples to show valid examples